app/templates/rounds/index.hbs

```diff
@@ -1,21 +1,5 @@
 {{#paper-list}}
   {{#each model as |round|}}
-    {{round-resource round=round}}
+    {{round-resource round=round goToRound=(action 'goToRound')}}
```

Comment:
https://github.com/frolfr/frolfr-web/pull/3/commits/31befd2d69fb1968e8910dabf5034c723121bfaf#diff-601bcde9e81ced6da325df4980ba5c94R16. This requires a parameter. How is this working if you're not passing in `round`?